build.sbt

```diff
@@ -1,9 +1,7 @@
-import Dependencies._
-
 ThisBuild / scalaVersion     := "2.13.2"
 ThisBuild / version          := "0.1.0-SNAPSHOT"
-ThisBuild / organization     := "com.example"
-ThisBuild / organizationName := "example"
+ThisBuild / organization     := "com.gu"
+ThisBuild / organizationName := "The Guardian"
```

Comment:
Do you need these unless you're building a library?

Reply:
No they are not needed if nothing is importing the classes.